gulpfile.js

```diff
@@ -65,6 +66,26 @@ var buildConfig = {
       vendor: {
         src: 'source/frontend/vendor/js/development/vendor.js',
```

Comment:
We can probably get rid of it, @greybubble.  The `bundles` property is what is governing the JS build now.